admin/src/main/resources/io/buoyant/admin/js/src/dashboard.js

```diff
@@ -0,0 +1,77 @@
+"use strict";
+
+define([
+  'jQuery', 'Handlebars', 'bootstrap',
+  'src/metrics_collector',
+  'src/routers',
+  'src/process_info',
+  'src/request_totals',
+  'src/router_controller'
+], function(
+  $, Handlebars, bootstrap,
+  MetricsCollector,
+  Routers,
+  ProcInfo,
+  RequestTotals,
+  RouterController
+) {
+  return function() {
+    /**
+     * Number of millis to wait between data updates.
+     */
+    var UPDATE_INTERVAL = 1000;
+
+    $.when(
+      $.get("/files/template/router_container.template"),
+      $.get("/files/template/router_server.template"),
+      $.get("/files/template/router_client.template"),
+      $.get("/files/template/router_client_container.template"),
+      $.get("/files/template/router_server_container.template"),
+      $.get("/files/template/server_rate_metric.partial.template"),
+      $.get("/files/template/metric.partial.template"),
+      $.get("/files/template/router_summary.template"),
+      $.get("/files/template/process_info.template"),
+      $.get("/files/template/request_totals.template"),
+      $.get("/admin/metrics.json")
```

Comment:
You're right! doing this means we can get rid of loading all the templates and passing them down from the router_controller too. I've moved templates/ to js/ templates and gotten rid of most of these $gets.

I would like to stop passing these down, and instead directly include the templates in the modules that they are used. This will involve a little refactoring (the module initializations) so I will make a branch right after this one, so the changes will actually be reflected in the diffs.